package.json

```diff
@@ -35,6 +55,8 @@
     "build:prod": "NODE_ENV=production parcel build src/index.html",
     "build:dev": "NODE_ENV=development parcel build src/index.html --no-minify",
     "dev": "parcel watch src/index.html",
+    "lint": "eslint '**/*.{js,jsx}' --quiet",
```

Comment:
This is different than the pre-commit hook, please unify as this is going to be used for CI verification.